src/schema/objects/entities.yaml

```diff
@@ -53,6 +46,14 @@ ceagent:
     `"ContrastBolusIngredient"` MAY also be added in the JSON file, with the same label.
   type: string
   format: label
+cohort:
+  name: cohort
+  display_name: Cohort
+  description: |
+    A subset of a defined template space, for instance, for a longitudinal template of brain development
+    where infants were participants were averaged at three, six, and twelve months old.
```

Comment:
@pwighton can you make a code suggestion here to replace/edit/improve the definition of cohorts? Thanks!